doc/whats_new.rst

```diff
@@ -25,6 +25,9 @@ New features
    - Added the :class:`ensemble.IsolationForest` class for anomaly detection based on
      random forests. By `Nicolas Goix`_.
 
+   - Added new supervised learning algorithm: :ref:`Multi-layer Perceptron <multi-layer-perceptron>`
```

Comment:
the correct target is ``multilayer_perceptron``, which is why the linking doesn't work. See https://raw.githubusercontent.com/scikit-learn/scikit-learn/master/doc/modules/neural_networks_supervised.rst